hack/lima-kubestellar.yaml

```diff
@@ -0,0 +1,179 @@
+# Please edit the following configuration for Lima instance "kubestellar"
+# and an empty file will abort the edit.
+
+# A template to install and use a KubeStellar demo environment with Lima.
+# $ limactl start lima-kubestellar.yaml --name kubestellar --vm-type=vz --mount-type=virtiofs --cpus 6 --memory 32 --tty=false
+# $ limactl shell kubectl --context its1 get managedclusters
+
+# To access KubeStellar on the host (assumes kubectl is installed), run the following commands:
```

Comment:
The meaning of the word "host" here is not entirely clear. See other comment about this.